controller.py

```diff
@@ -0,0 +1,30 @@
+from view import *
+
+from model import *
+
+class Controller:
+
+    def __init__(self,path):
+        self.shortlist = load_shortlist(path)
+        self.applicants = load_applicants(path)
+        self.view = View()
+
+    def show_applicant_info(self):
+        self.view.view_applicants(self.shortlist.applicants)
+    
+    def show_applicants(self):
+        self.view.view_applicants(self.applicants)
+    
+    def show_role_info(self):
+        self.view.view_role(self.shortlist.role)
+
+    def show_shortlist(self):
+        self.view.view_shortlist(self.shortlist)
+
+"""job = Role("employee","18230123",[])
+view = View()"""
```

Comment:
It's good to get rid of comments like these - things are are definitely no longer going to be part of the code.